src/js/worklets/onboarding_carousel.js

```diff
@@ -1,8 +1,8 @@
 import { useDerivedValue, withTiming, Easing } from 'react-native-reanimated';
 
 const slideAnimationDuration = 300;
-const totalPages = 4;
-const pageSize = 100 / totalPages;
```

Comment:
I tried using 100/3, but 33.333333 is causing issues in some cases. (I will improve this further while refactoring the carousel-related code.)